tests/apps/julia/julia-linalg.py

```diff
@@ -0,0 +1,80 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+class JuliaLinalgBaseTest(rfm.RunOnlyRegressionTest):
+    def __init__(self):
+        self.valid_prog_environs = ['builtin']
+        self.sanity_patterns = sn.assert_found(r'Julia version:*',
+                                               self.stdout)
+        self.modules = ['Julia']
+        self.executable = 'julia'
+        self.executable_opts = ['linalg.jl']
+        self.tags = {'apps', 'julia', '1nodes'}
+        self.maintainers = ['Lewih']
+
+
+@rfm.simple_test
+class JuliaLinalgTest(JuliaLinalgBaseTest):
+    def __init__(self):
+        super().__init__()
+        self.descr = 'Test a few typical Julia LinAlg operations'
+        self.valid_systems = ['*:single-node']
+        self.num_tasks_per_node = 1
+        self.num_cpus_per_task = 20
+        self.tags.add('performance')
+        self.executable_opts += [str(self.num_cpus_per_task)]
+        self.variables = {"JULIA_DEPOT_PATH": "$VSC_SCRATCH/rfm_julia_$SLURM_JOBID"}
+        self.postrun_cmds = ['rm -rf $VSC_SCRATCH/rfm_julia_$SLURM_JOBID']
```

Comment:
Note that `SLURM_JOBID` is not defined on clusters that do not use the Slurm scheduler (such as Genius). For this test this probably not an issue, unless several of these tests would run concurrently.

Reply:
Oi oi oi, I fix it asap